support-frontend/assets/helpers/tracking/googleTagManager.js

```diff
@@ -202,14 +202,15 @@ function successfulConversion(participations: Participations) {
   sendData('SuccessfulConversion', participations);
 }
 
-function gaEvent(gaEventData: GaEventData) {
+function gaEvent(gaEventData: GaEventData, additionalFields: ?Object) {
   maybeTrack(() => {
     if (window.googleTagManagerDataLayer) {
       window.googleTagManagerDataLayer.push({
         event: 'GAEvent',
         eventCategory: gaEventData.category,
         eventAction: gaEventData.action,
         eventLabel: gaEventData.label,
+        ...additionalFields,
```

Comment:
What would this object look like in practice out of curiosity?

Reply:
```javascript
{
    event: "GAEvent", 
    eventCategory: "click", 
    eventAction: "payment-method-selected", 
    eventLabel: "Stripe", 
    paymentMethod: "Stripe"
}
```